app/models/course.rb

```diff
@@ -180,8 +186,15 @@ def show_stdout_and_stderr=(option)
   def set_defaults
     self.start_at ||= Time.zone.now.beginning_of_hour
     self.end_at ||= self.start_at + 1.month
+    self.default_reference_timeline ||= reference_timelines.new(default: true)
 
     return unless creator && course_users.empty?
     course_users.build(user: creator, role: :owner, creator: creator, updater: updater)
   end
+
+  def validate_only_one_default_reference_time
+    num_defaults = reference_timelines.where(course_reference_timelines: { default: true }).count
+    return if num_defaults <= 1 # Could be 0 if item is new
+    errors.add(:reference_timelines, 'cannot have more than 1 default')
```

Comment:
This should be a translation. See https://github.com/Coursemology/coursemology2/blob/master/app/models/course/assessment/answer.rb#L131 and https://github.com/Coursemology/coursemology2/blob/master/config/locales/en/activerecord/course/assessment/answer.yml#L9 for an example.